### PR TITLE
Partial CI fixes

### PR DESCRIPTION
**Pull request by nabobalis**
Trying to pass what is failing on main.

### CODE REVIEW

File: sunpy/image/transform.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def affine_transform(image, rmatrix, order=3, scale=1.0, image_center=None,
         Defaults to `True` i.e., recenter to the center of the array.
     missing : `float`, optional
         The value to replace any missing data after the transformation.
-    method : {'skimage', 'scipy'}
+    method : {``'skimage'``, ``'scipy'``}, optional
```
Comment (nabobalis) [inline sunpy/image/transform.py:43]: Not sure if this is the sane way to fix this. I need to double check some docs.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -107,7 +107,7 @@ stages:
           submodules: false
           coverage: codecov
           toxdeps: tox-pypi-filter
-          posargs: -n auto --dist loadgroup
+          posargs: -n auto --dist loadscope
```
Comment (nabobalis) [inline azure-pipelines.yml:110]: not sure if this is the correct one.

Comment (dstansby) [reply azure-pipelines.yml:110]: Use `loadfile`, it's what I did in GH actions

Comment (ConorMacBride) [reply azure-pipelines.yml:110]: This is updating the cron stage although I think only oldestdeps should need it? Could just update SecondPhaseTests with
```yaml
- linux: py38-oldestdeps
  posargs: -n auto --dist loadfile
```

Comment (nabobalis) [reply azure-pipelines.yml]: Woops, clearly was paying no attention to what block I was editing.

**Comment by nabobalis** [comment]
Doc CI fail is 5 failed examples.

**Comment by ConorMacBride** [comment]
For the cron test there's a missing time unit warning in sunpy/net/tests/strategies.py. And there's some other warnings in devdeps for 3.1 and 3.0. I can take a look tomorrow if you like.

Not sure what to do with pyinstaller. Do we need it?

**Comment by nabobalis** [comment]
I will have a look at those in a separate PR.

**Review by ConorMacBride — APPROVED** [review]

**Comment by nabobalis** [comment]
The online tests had a block of connection issues:
```
FAILED ../../.tox/py39-online/lib/python3.9/site-packages/sunpy/database/tests/test_database.py::test_hek_query_download
FAILED ../../.tox/py39-online/lib/python3.9/site-packages/sunpy/database/database.py::sunpy.database.database.Database.fetch
FAILED ../../.tox/py39-online/lib/python3.9/site-packages/sunpy/database/tests/test_database.py::test_vso_query_block_caching
FAILED ../../.tox/py39-online/lib/python3.9/site-packages/sunpy/database/tests/test_database.py::test_vso_query_block_caching_with_overwrite_true_flag
FAILED ../../.tox/py39-online/lib/python3.9/site-packages/sunpy/database/tests/test_database.py::test_download_from_qr
FAILED ../../.tox/py39-online/lib/python3.9/site-packages/sunpy/database/tests/test_database.py::test_fetch_separate_filenames
FAILED ../../.tox/py39-online/lib/python3.9/site-packages/sunpy/net/vso/tests/test_vso.py::test_path
```
I think tis fine to merge.

**Comment by ConorMacBride** [comment]
The change in azure-pipelines.yml doesn't need backported

**Comment by nabobalis** [comment]
I probably end up doing one set of backports for each branch when the time comes.

**Pull request merged**